draft/mson-parameters-headers.md

```diff
@@ -0,0 +1,216 @@
+---
+RFC: XXXX
+Author: Z
+Status: Draft
+Created: 2015-09-23
+Last Modified: 2015-09-28
+---
+
+# API Blueprint RFC XXXX: MSON Parameters and Headers
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Implementation](#implementation)
+- [Backwards Compatibility](#backwards-compatibility)
+
+## Abstract
+
+This RFC proposes using full [MSON][] syntax for description of URI parameters
+and HTTP headers.
+
+## Motivation
+
+Using MSON syntax for description of URI parameters and HTTP Headers will
+greatly help to unify the API Blueprint syntax and introduce reusability of
+URI parameters and HTTP headers. It will also greatly introduce the reusability
```

Comment:
@geemus thanks I think my head was lost somewhere between the copy and paste 🙃 Any overall thoughts on this?